disconnected-operators.sh

```diff
@@ -41,11 +37,20 @@ oc adm catalog build \
 oc patch OperatorHub cluster --type json \
   -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
 
-oc adm catalog mirror \
+oc adm catalog mirror --manifests-only \
```

Comment:
So this only calculates the manifests required for mirroring, but do not actually mirror image content. What does it actually do in Docker registry?

Reply:
I see, it generates manifests locally (`./redhat-operators-manifests/`)